scripts/test-extensions.bat

```diff
@@ -8,23 +8,54 @@ set VSCODEUSERDATADIR=%TMP%\adsuser-%RANDOM%-%TIME:~6,5%
 set VSCODEEXTENSIONSDIR=%TMP%\adsext-%RANDOM%-%TIME:~6,5%
 echo %VSCODEUSERDATADIR%
 echo %VSCODEEXTENSIONSDIR%
+
+:: Default to only running stable tests if test grep isn't set
+if "%ADS_TEST_GREP%" == "" (
+	echo Running stable tests only
+	set ADS_TEST_GREP=@UNSTABLE@
+	SET ADS_TEST_INVERT_GREP=1
+)
+
 @echo OFF
 
-echo starting admin tool extension windows tests
+echo ***************************************************
+echo *** starting admin tool extension windows tests ***
+echo ***************************************************
 call .\scripts\code.bat --nogpu --extensionDevelopmentPath=%~dp0\..\extensions\admin-tool-ext-win --extensionTestsPath=%~dp0\..\extensions\admin-tool-ext-win\out\test --user-data-dir=%VSCODEUSERDATADIR% --extensions-dir=%VSCODEEXTENSIONSDIR% --disableExtensions --remote-debugging-port=9222
-echo starting agent tests
+
+echo ****************************
+echo *** starting agent tests ***
+echo ****************************
 call .\scripts\code.bat --nogpu --extensionDevelopmentPath=%~dp0\..\extensions\agent --extensionTestsPath=%~dp0\..\extensions\agent\out\test --user-data-dir=%VSCODEUSERDATADIR% --extensions-dir=%VSCODEEXTENSIONSDIR% --remote-debugging-port=9222
-echo starting azurecore tests
+
+echo ********************************
+echo *** starting azurecore tests ***
+echo ********************************
 call .\scripts\code.bat --nogpu --extensionDevelopmentPath=%~dp0\..\extensions\azurecore --extensionTestsPath=%~dp0\..\extensions\azurecore\out\test --user-data-dir=%VSCODEUSERDATADIR% --extensions-dir=%VSCODEEXTENSIONSDIR% --remote-debugging-port=9222
-echo starting cms tests
+
+echo **************************
+echo *** starting cms tests ***
+echo **************************
 call .\scripts\code.bat --nogpu --extensionDevelopmentPath=%~dp0\..\extensions\cms --extensionTestsPath=%~dp0\..\extensions\cms\out\test --user-data-dir=%VSCODEUSERDATADIR% --extensions-dir=%VSCODEEXTENSIONSDIR% --remote-debugging-port=9222
-echo starting dacpac tests
-call .\scripts\code.bat --extensionDevelopmentPath=%~dp0\..\extensions\dacpac --extensionTestsPath=%~dp0\..\extensions\dacpac\out\test --user-data-dir=%VSCODEUSERDATADIR% --extensions-dir=%VSCODEEXTENSIONSDIR% --remote-debugging-port=9222
-echo starting schema compare tests
+
+echo *****************************
+echo *** starting dacpac tests ***
+echo *****************************
+call .\scripts\code.bat --nogpu --extensionDevelopmentPath=%~dp0\..\extensions\dacpac --extensionTestsPath=%~dp0\..\extensions\dacpac\out\test --user-data-dir=%VSCODEUSERDATADIR% --extensions-dir=%VSCODEEXTENSIONSDIR% --remote-debugging-port=9222
```

Comment:
oops - forgot to undo this change. But @aaomidi is there a reason you don't have this set for dacpac tests? They seem to run fine with it

Reply:
You mean nogpu? No reason I may have just forgot.

Reply:
Alright, I'll leave it in and see if it breaks anything =)